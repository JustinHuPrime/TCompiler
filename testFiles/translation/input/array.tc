module foo;

int[3] bar = [31, 41, 59];
