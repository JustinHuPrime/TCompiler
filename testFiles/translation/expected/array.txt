testFiles/translation/input/array.tc:
DATA(_T3foo3bar, 4,
  INT(31),
  INT(41),
  INT(59),
)
